src/Components/ProgressBar.jsx

```diff
@@ -1,34 +1,36 @@
 import "../CSS/Calculator.css";
 
-const ProgressBar = ({ currentSectionName, currentSection, totalSections }) => {
+const ProgressBar = ({ currentSectionName, totalSections }) => {
   let sectionName;
-  let barWidthCurrent = `${Math.floor(
-    (currentSection / totalSections) * 100
-  )}%`;
-  let barWidthActive = `${Math.floor(
-    ((currentSection - 1) / totalSections) * 100
-  )}%`;
-  let stepArr = [];
-  let stepWidth = `${100 / totalSections - 4.5}%`;
+  let currentSection;
+  let stepWidth = `29%`;
+  let stepBackground = "#e5e5e5";
+  let stepBackgroundActive = "#4E6C99";
 
   switch (currentSectionName) {
     case "conv":
       sectionName = "Your Conviction";
+      currentSection = 1;
       break;
     case "circ":
       sectionName = "Surrounding Circumstances";
+      currentSection = 2;
       break;
     case "ter":
       sectionName = "Terms of Offence";
+      currentSection = 3;
+      break;
+    case "conv-1" || "circ-1" || "ter-1":
+      currentSection = 4;
```

Comment:
Is there supposed to be no section name if it's one of these 3 sections?

Reply:
Yep, the Figma file (as of now) doesn't have a name for the eligible page but it does have a completed progress bar